internal/clusterconfig/clusterconfig_test.go

```diff
@@ -266,6 +274,7 @@ func TestClusterConfig(t *testing.T) {
 				if err := utils.ReadYamlOrJSONFile(filepath.Join(clusterConfigPath, common.SeedClusterInfoFileName), seedReconfig); err != nil {
 					t.Errorf("unexpected error: %v", err)
 				}
+				assert.Equal(t, "$2a$10$20Q4iRLy7cWZkjn/D07bF.RZQZonKwstyRGH0qiYbYRkx5Pe4Ztyi", seedReconfig.KubeadminPasswordHash)
```

Comment:
Add a test for kubeadmin secret not found

Reply:
Created #287 before adding test